src/Sylius/Bundle/AdminBundle/Resources/views/Promotion/_form.html.twig

```diff
@@ -40,4 +40,3 @@
         </div>
     </div>
 </div>
-
```

Comment:
No, because in this PR with fixes associated with promotion, I remove also this unnecessary blank line 😃